pkgs/tools/filesystems/bcachefs-tools/default.nix

```diff
@@ -43,14 +43,6 @@ stdenv.mkDerivation (finalAttrs: {
     makeWrapper
   ];
 
-  cargoRoot = "rust-src";
-  cargoDeps = rustPlatform.importCargoLock {
-    lockFile = ./Cargo.lock;
-    outputHashes = {
-      "bindgen-0.64.0" = "sha256-GNG8as33HLRYJGYe0nw6qBzq86aHiGonyynEM7gaEE4=";
```

Comment:
I tried building with `${src}/Cargo.lock`, didn't work as it would fail to find the file.

Removing `importCargoLock` will fail to build the rust code.

Reply:
We could use `cargoHash`.

Reply:
> We could use `cargoHash`.

Doesn't look like it's working.
```
┃        > unpacking source archive /nix/store/qyrbgxh64055v3v93dbasp8x6s2xhg4r-source
┃        > source root is source
┃        > Executing cargoSetupPostUnpackHook
┃        > unpacking source archive
┃        > do not know how to unpack source archive
```

Reply:
> I tried building with `${src}/Cargo.lock`, didn't work as it would fail to find the file.

Nevermind, I should have used `finalAttrs.src`. Successfully built. Pushing.

Reply:
Looks like ofborg eval is failing with latest changes. Successfully builds though.

Reply:
It's failing because it's import-from-derivation, which is not allowed in Nixpkgs.

> We are not using `rustPlatform.buildRustPackage` to build a "rust-only package" here.

Ah, then what we need is `rustPlatform.fetchCargoTarball`.

Reply:
Nice, Ofborg eval working with fetchCargoTarball.
As for the update script, is there a script for automatically updating `hash` in `fetchCargoTarball`?